crates/forge-runner/src/gas.rs

```diff
@@ -138,6 +143,30 @@ fn get_state_resources(
     })
 }
 
+fn reduce_state_resources_after_strk_token_predeployment(
+    state_resources: &mut StateResources,
+) -> StateResources {
+    // STRK predeployment results in state changes. To avoid including them in gas cost
+    // we need to reduce the state changes count. These calculations could be slightly inaccurate only if
+    // someone would modify storage cells which are changed in the STRK constructor.
+
+    state_resources
+        .state_changes_for_fee
+        .state_changes_count
+        .n_storage_updates -= 10;
+    state_resources
+        .state_changes_for_fee
+        .state_changes_count
+        .n_class_hash_updates -= 1;
+    state_resources
+        .state_changes_for_fee
+        .state_changes_count
+        .n_modified_contracts -= 1;
+    state_resources.state_changes_for_fee.n_allocated_keys -= 10;
+
+    state_resources.clone()
+}
+
```

Comment:
note: As mentioned in the comment, these calculations could be inaccurate only if user updated storage cells which are modified during execution of [STRK constructor](https://github.com/starknet-io/starkgate-contracts/blob/2c2729d47b9b5c7421c16fd258b2fa7e3ad6ddfc/src/cairo/strk/erc20_lockable.cairo#L125).